latexify/core.py

```diff
@@ -70,34 +61,7 @@ def visit_Set(self, node):
     return r'\left\{ ' + r'\space,\space '.join([self.visit(i) for i in node.elts]) + r'\right\} '
 
   def visit_Call(self, node):
-    builtin_callees = {
-        'abs': (r'\left|{', r'}\right|'),
-        'math.acos': (r'\arccos{\left({', r'}\right)}'),
-        'math.acosh': (r'\mathrm{arccosh}{\left({', r'}\right)}'),
-        'math.asin': (r'\arcsin{\left({', r'}\right)}'),
-        'math.asinh': (r'\mathrm{arcsinh}{\left({', r'}\right)}'),
-        'math.atan': (r'\arctan{\left({', r'}\right)}'),
-        'math.atanh': (r'\mathrm{arctanh}{\left({', r'}\right)}'),
-        'math.ceil': (r'\left\lceil{', r'}\right\rceil'),
-        'math.cos': (r'\cos{\left({', r'}\right)}'),
-        'math.cosh': (r'\cosh{\left({', r'}\right)}'),
-        'math.exp': (r'\exp{\left({', r'}\right)}'),
-        'math.fabs': (r'\left|{', r'}\right|'),
-        'math.factorial': (r'\left({', r'}\right)!'),
-        'math.floor': (r'\left\lfloor{', r'}\right\rfloor'),
-        'math.fsum': (r'\sum\left({', r'}\right)'),
-        'math.gamma': (r'\Gamma\left({', r'}\right)'),
-        'math.log': (r'\log{\left({', r'}\right)}'),
-        'math.log10': (r'\log_{10}{\left({', r'}\right)}'),
-        'math.log2': (r'\log_{2}{\left({', r'}\right)}'),
-        'math.prod': (r'\prod \left({', r'}\right)'),
-        'math.sin': (r'\sin{\left({', r'}\right)}'),
-        'math.sinh': (r'\sinh{\left({', r'}\right)}'),
-        'math.sqrt': (r'\sqrt{', '}'),
-        'math.tan': (r'\tan{\left({', r'}\right)}'),
-        'math.tanh': (r'\tanh{\left({', r'}\right)}'),
-        'sum': (r'\sum \left({', r'}\right)'),
-    }
+    builtin_callees = constants.BUILTIN_CALLEES
```

Comment:
@odashi @ShigekiKarita 
Should I declare a local variable here to access it more simply in the function definition (line 67 - 68), or not (i.e. keep using `constants.BUILTIN_CALLEES` throughout the definition)?

In my understanding functionality remains the same.

Reply:
Usually I define aliases if something will be used many times. In this case it's basically up to you, but I think using `dict.get` seems more reasonable to avoid alias:

```python
lstr, rstr = constants.BUILTIN_CALLEES.get(callee_str, (None, None))
if lstr is None:
  ...
```

Reply:
I made the suggested change.